packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -125,6 +126,11 @@ export interface GridOptions {
    * @default false
    */
   disableSelectionOnClick?: boolean;
+  /**
+   * Controls whether to use the cell or row editing.
+   * @default "cell"
+   */
+  editMode?: 'row' | 'cell';
```

Comment:
We could introduce a variable `GridEditMode` since it's used 5 times

Reply:
I added a `GridEditMode` type alias.